gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/auth/AllowAllAuthenticator.java

```diff
@@ -41,7 +42,16 @@ public AuthenticatedUser authenticate(final Map<String, String> credentials) thr
         return AuthenticatedUser.ANONYMOUS_USER;
     }
 
+    /**
+     * @deprecated As of release 3.1.1-incubating, replaced by {@link #newSaslNegotiator(InetAddress)}.
+     * @see <a href="https://issues.apache.org/jira/browse/TINKERPOP3-995">TINKERPOP3-995</a>
```

Comment:
This is about as tiny as it comes, but could the '3' be dropped to be consistent with the latest name update?  Can't remember if they were ever going to drop the redirect.  I'm sure there are others in the code so maybe instead, at some point a mass update could be made if we wanted.